lang/ftidy/Portfile

```diff
@@ -1,20 +1,21 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem              1.0
+PortGroup               compilers 1.0
 
 name                    ftidy
 version                 7.2
-revision                1
+revision                2
 categories              lang
 platforms               darwin
 license                 Noncommercial
 maintainers             nomaintainer
 homepage                http://web.archive.org/web/20120905025547/http://www.unb.ca/fredericton/science/chem/ajit/tidy/
-master_sites            ${homepage}
-distname                tidy72sc
+master_sites            http://cd.textfiles.com/simtel/simtel0101/simtel/fortran
+distname                tidy72
```

Comment:
Why do you want to change the distfile? The contents of tidy72.zip are identical to tidy72sc.zip except that tidy72.zip is more than twice the size because it includes tidy.exe which we don't need. So I recommend reverting your changes to the distname, master_sites, checksums.

Reply:
OK